cli/src/cmd/forge/script/sequence.rs

```diff
@@ -241,33 +241,33 @@ pub struct TransactionWithMetadata {
     pub hash: Option<TxHash>,
     #[serde(rename = "type")]
```

Comment:
Doesn't this one need to be changed as well?

Reply:
You are absolutely right, I missed this